examples/python/queries/accelerated_queries.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+import _dummy_objects
+
+# Load a map
+your_map = _dummy_objects.example_map()
+
+# Vectorized query for a list of fixed-resolution indices
```

Comment:
Maybe update the comments to be a bit more descriptive. The first one queries at whatever resolution the index is at while the second one specified the height/resolution per query index correct?

Reply:
Done :)